pipelines/test/tw/staticpackage.yaml

```diff
@@ -0,0 +1,12 @@
+name: Static Package Check
+
+needs:
+  packages:
+    - package-type-check
+
+pipeline:
+  - name: Check if the package is a Static Package
+    runs: |
+      pkg="${{context.name}}"
+
+      package-type-check static "$pkg"
```

Comment:
skip the use of the shell var and just:

```package-type-check static "${{context.name}}"```

Unless I'm missin something, and there is a reason for the intermediate shell var.  other tw/ pipelines use melange variables directly.